third_party/libosmium/test/t/io/data-cr.opl

```diff
@@ -1 +1 @@
-n1 v1 dV c1 t2014-01-01T00:00:00Z i1 utest T x1.02 y1.02
+n1 v1 dV c1 t2014-01-01T00:00:00Z i1 utest T x1.02 y1.02
```

Comment:
@daniel-j-h is this an unrelated change?

Reply:
@mokob it is fixed already in upstream master.